tests/data/output_schema_as_JSON_schema.yaml

```diff
@@ -0,0 +1,47 @@
+title: An example Pipestat output schema
+description: A pipeline that uses pipestat to report sample and project level results.
+type: object
+properties:
+  pipeline_name: "default_pipeline_name"
+  samples:
+    type: array
+    items:
+      type: object
+      properties:
+        number_of_things:
+          type: integer
+          description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+        smooth_bw:
+          type: string
+          description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce nec cursus nulla."
+          path: "aligned_{genome}/{sample_name}_smooth.bw"
+        collection_of_images:
+          type: array
+          description: A collection of images.
+          items:
+            type: object
+            properties:
+              prop1:
+                type: file
+                description: An example file.
+        output_file_in_object:
+          type: object
+          description: An object containing output files.
+          properties:
+            example_property_1:
+              type: file
+              description: An example file.
+            example_property_2:
+              type: image
+              description: An example image.
+  project:
+    type: array
```

Comment:
Why is this an array of objects, instead of just an object?

Reply:
I can double check but I believe it could be either. I used array based on some example reading I found.

Reply:
I decided to change this to object instead of array. I believe it will make the schema easier for the end user to write and read.